interpreter/cling/tools/plugins/clad/CMakeLists.txt

```diff
@@ -20,6 +20,12 @@ if("${CMAKE_GENERATOR}" STREQUAL "Ninja")
   )
 endif()
 
+if (builtin_clang)
```

Comment:
@chrisburr, could you move this code here: https://github.com/root-project/root/blob/master/interpreter/CMakeLists.txt#L415-L420 -- this way it can be used in other places, too.